engine/crates/federated-graph/src/from_sdl.rs

```diff
@@ -96,38 +107,31 @@ impl<'a> State<'a> {
         StringId(self.strings.insert_full(s.to_owned()).0)
     }
 
-    fn insert_value(&mut self, node: &async_graphql_value::ConstValue, expected_enum_type: Option<EnumId>) -> Value {
+    fn insert_value(&mut self, node: ast::Value<'_>, expected_enum_type: Option<EnumId>) -> Value {
         match node {
-            async_graphql_value::ConstValue::Null => Value::String(self.insert_string("null")),
-            async_graphql_value::ConstValue::Number(number) => {
-                if let Some(number) = number.as_i64() {
-                    Value::Int(number)
-                } else if let Some(number) = number.as_f64() {
-                    Value::Float(number)
-                } else {
-                    unreachable!()
-                }
-            }
-            async_graphql_value::ConstValue::String(s) => Value::String(self.insert_string(s)),
-            async_graphql_value::ConstValue::Boolean(b) => Value::Boolean(*b),
-            async_graphql_value::ConstValue::Enum(enm) => expected_enum_type
+            ast::Value::Null => Value::String(self.insert_string("null")),
```

Comment:
Are we representing nulls as a string?  That seems... odd.

Reply:
Indeed, not from this PR but I can't remember a good reason to do that. I'll look into fixing that.